pkg/secrets/provide_conjur_secrets.go

```diff
@@ -97,3 +96,51 @@ func RetryableSecretProvider(
 		return err
 	}
 }
+
+var TickerQuit = make(chan bool)
+var ProviderDone = make(chan bool)
+func PeriodicSecretProvider(
+	secretRefreshInterval time.Duration,
+	mode string,
+	provideSecrets ProviderFunc,
+	timeout time.Duration,
+	) ProviderFunc {
+	return func() error {
+		err := provideSecrets()
+		switch {
+		case mode == "init":
+			return err
+		case mode == "sidecar" && secretRefreshInterval > 0:
+			ticker := time.NewTicker(secretRefreshInterval)
+			go periodicSecretProvider(provideSecrets, ticker)
+		default:
+			// fall through
+		}
+		// Wait here
+		select {
+		case <-ProviderDone: // Todo change to SIGTERM or SIGINTERRUPT
+		case <-time.After(timeout):
```

Comment:
Can we just delete this line?:
```
    case <-time.After(timeout):
```
the previous case `case <-ProviderDone:` should be enough to wait forever, right?

Reply:
deleted.